src/GUI/ParametricFunctionControl.java

```diff
@@ -39,6 +41,18 @@ public void actionPerformed(ActionEvent e) {
                 graphPanel.cameraReset();
             }
         }
+        public class ValueTableActionListener implements ActionListener {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                ParametricExpression expression = graphPanel.getParametricExpression();
+                if (expression != null) {
+                    Valuetable valuetable = new Valuetable(expression);
```

Comment:
Variable valuetable is never used anywhere else. So new Valuetable would be sufficient.